cmd/prepare-storage.go

```diff
@@ -217,7 +227,11 @@ func waitForFormatXL(ctx context.Context, firstDisk bool, endpoints EndpointList
 				switch err {
 				case errNotFirstDisk:
 					// Fresh setup, wait for first server to be up.
-					logger.Info("Waiting for the first server to format the disks.")
+					logger.Info("Waiting for the first server to be online to format the disks.")
```

Comment:
Should be `Waiting for the first server to format the disks`

because first server can be online but waiting on one of the other disks to come online so that it can format.